### PR TITLE
Updating Raspbian installation for Raspbian Buster

### PR DESCRIPTION
**Pull request by franklesniak**
Raspbian Buster was released alongside the Raspberry Pi 4. Buster does not include libssl1.0, which is a prerequisite for .NET Core 2.x. As a result, PowerShell Core 6.x would fail. Therefore, libssl1.0 is added as a prerequisite with care to not select libssl1.0-dev. Also, a step was added to update package lists prior to package installation, and the -y flag on the apt-get install command prevents a confirmation prompt.

Note that libssl1.0 is not required for PowerShell 7 because PS7 runs on .NET Core 3.x, which does not have the dependency.

Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [X] This issue only shows up in version 6.x of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by franklesniak** [comment]
The step to update package lists:

# Update package lists
sudo apt-get update

and the -y flag on the apt-get install command should be added to PowerShell v7's instructions. I intended to submit a PR for that version of the doc, but I'm too much of a noob to figure out how - or maybe I do not have permissions to view the v7 branch of the doc.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -606,8 +606,18 @@ Download [Raspbian Stretch](https://www.raspberrypi.org/downloads/raspbian/) and
...
-sudo apt-get install libunwind8
+###################################
+# Prerequisites
+
+# Update package lists
+sudo apt-get update
+
+# Install libunwind8 and libssl1.0
+# Regex is used to ensure that we do not install libssl1.0-dev, as it is a variant that is not required
+sudo apt-get install '^libssl1.0(.?.?.?|.*([^v]|[^e]v|[^d]ev|[^-]dev))$' libunwind8 -y
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: why do we need the regex at all?

Comment (franklesniak) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: Hey Travis, I explored the option of just running:
`sudo apt-get install libssl1.0 libunwind8`
but apt-get inherently uses regex when it does not find an exact version match. So in this case, _libssl1.0_ matched to:

- libssl1.0.0
- libssl1.0.2, and
- libssl1.0-dev

libssl1.0.0 and 1.0.2 essentially are the same thing, and they are what we are trying to ensure is installed as a prerequisite. libssl1.0-dev is a different package that is not needed. In plain English, the regex expression is saying "give me the latest version of libssl1.0.x, but do not install libssl1.0-dev"

Hope that helps clarify. Thanks!

Comment (TravisEz13) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: wouldn't this work better?
```suggestion
sudo apt-get install '^libssl1.0.[0-9]$' libunwind8 -y
```

Comment (franklesniak) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: This looks reasonable to me. Let me test it out on my Pi4 to make sure there's no issues with Raspbian Buster and I'll report back.

Comment (franklesniak) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: OK, your suggestion worked well on Raspbian Buster. See screenshot:
2019-07-10-213636_3840x2160_scrot_cropped

**Comment by sdwheeler** [comment]
@franklesniak Thanks for the change. However, in the future, please only submit changes to the `staging` branch. I didn't notice that this change was targeted at the `live` branch. 

NOTE: Changes made in the `staging` branch get merged into `live` so your change would be overwritten. I have copied your changes to the `staging` branch.

**Comment by franklesniak** [comment]
> @franklesniak Thanks for the change. However, in the future, please only submit changes to the staging branch. I didn't notice that this change was targeted at the live branch.
> NOTE: Changes made in the staging branch get merged into live so your change would be overwritten. I have copied your changes to the staging branch.

Ah, I was wondering about that... as a noob, it was not clear to me where to submit. Thanks for the clarification!

**Comment by sdwheeler** [comment]
@franklesniak No worries. You help me find a gap in our Contributor Guide. I have updated it. See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/7-PULL-REQUESTS.md

**Comment by franklesniak** [comment]
> @franklesniak No worries. You help me find a gap in our Contributor Guide. I have updated it. See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/7-PULL-REQUESTS.md

Looks good! Clear and concise. Thanks for doing that.
